desktop/core/src/desktop/settings.py

```diff
@@ -334,8 +334,13 @@
     'DEFAULT_PERMISSION_CLASSES': [
       'rest_framework.permissions.IsAuthenticated',
     ],
-    'DEFAULT_AUTHENTICATION_CLASSES': desktop.conf.AUTH.API_AUTH.get()
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+          'rest_framework_simplejwt.authentication.JWTAuthentication',
+          'rest_framework.authentication.SessionAuthentication'
```

Comment:
If people want to use their own auth, we might want to keep the DEFAULT_AUTHENTICATION_CLASSES param so that they can override the full list too (might not be used right away, but at least we don't hardcode it)